src/main/java/com/gluonhq/substrate/model/InternalProjectConfiguration.java

```diff
@@ -376,8 +377,7 @@ public IosSigningConfiguration getIosSigningConfiguration() {
      * @throws IllegalArgumentException when the configuration doesn't contain a property graalPath
      * @throws IOException when the path to bin/native-image doesn't exist
      */
-    public void canRunNativeImage() throws IOException {
-
+    public void canRunNativeImage() throws IOException, InterruptedException {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [The method 'canRunNativeImage()' has an NPath complexity of 224](https://app.codacy.com/manual/gluonhq/substrate/pullRequest?prid=4983430)